From 4b8938d603180fdc7dbf4007bd0a57a08292436f Mon Sep 17 00:00:00 2001
From: Richard Zhu <hongxing.zhu@nxp.com>
Date: Mon, 5 Sep 2016 16:04:47 +0800
Subject: [PATCH 1153/5242] MLK-13186-1 ARM: imx: enable pcie external osc on
 imx6qp

commit  ed91cf0002f0a32fa5feea7a144e0e14e5a0c319 from
https://source.codeaurora.org/external/imx/linux-imx.git

In order to pass the pcie gen2 electronic compliance
tests, the external oscillator is manatory required.
Enable pcie external osc support on imx6qp platforms

Signed-off-by: Richard Zhu <Richard.Zhu@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qp.dtsi |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/imx6qp.dtsi b/arch/arm/boot/dts/imx6qp.dtsi
index 79d22b4..d378c33 100644
--- a/arch/arm/boot/dts/imx6qp.dtsi
+++ b/arch/arm/boot/dts/imx6qp.dtsi
@@ -46,10 +46,10 @@
 			                <0 0 0 2 &intc 0 122 IRQ_TYPE_LEVEL_HIGH>,
 			                <0 0 0 3 &intc 0 121 IRQ_TYPE_LEVEL_HIGH>,
 			                <0 0 0 4 &intc 0 120 IRQ_TYPE_LEVEL_HIGH>;
-			clocks = <&clks IMX6QDL_CLK_PCIE_REF_125M>,
-				 <&clks IMX6QDL_CLK_SATA_REF_100M>,
+			clocks = <&clks IMX6QDL_CLK_PCIE_REF_125M>, <&clks IMX6QDL_PLL6_BYPASS>,
+				 <&clks IMX6QDL_PLL6_BYPASS_SRC>,
 				 <&clks IMX6QDL_CLK_LVDS1_GATE>, <&clks IMX6QDL_CLK_PCIE_AXI>;
-			clock-names = "pcie_phy", "ref_100m", "pcie_bus", "pcie";
+			clock-names = "pcie_phy", "pcie_ext", "pcie_ext_src", "pcie_bus", "pcie";
 			status = "disabled";
 		};
 
-- 
1.7.9.5

